source/code/plugin/CAdvisorMetricsAPIClient.rb

```diff
@@ -13,7 +13,7 @@ class CAdvisorMetricsAPIClient
             require_relative 'KubernetesApiClient'
     
             @LogPath = "/var/opt/microsoft/omsagent/log/kubernetes_perf_log.txt"
```

Comment:
Can you update the file path here to var/opt/microsoft/docker-cimprov/ similar to /var/opt/microsoft/docker-cimprov/log/kubernetes_client_log.txt . Trying to remove dependency from omsagent directories as much as we can.

Reply:
will fix in the next round.